core/arch/arm/kernel/boot.c

```diff
@@ -1221,8 +1221,49 @@ void __weak boot_init_primary_late(unsigned long fdt __unused,
 	IMSG("Primary CPU switching to normal world boot");
 }
 
+#ifdef CFG_CORE_HALT_CORES_ON_PANIC
+
+#ifdef ARM32
+#define ARM_CNTXCTL_IMASK	BIT(1)
+
+static void mask_timer(void)
+{
+	/* Mask timer interrupts */
+	write_cntp_ctl(read_cntp_ctl() | ARM_CNTXCTL_IMASK);
+	write_cntv_ctl(read_cntv_ctl() | ARM_CNTXCTL_IMASK);
+}
+#endif /* ARM32 */
+
+static enum itr_return
+	multi_core_panic_it_handler(struct itr_handler *hdl  __unused)
```

Comment:
remove this tab
I think this function needs a `__noreturn` attribute

Reply:
No need to indent this line.